src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAddAudioMetadataIntTest.java

```diff
@@ -146,7 +146,7 @@ void addAudioMetadataNonExistingCourthouse() throws Exception {
 
         String actualJson = mvcResult.getResponse().getContentAsString();
         String expectedJson = """
-            {"type":"COMMON_100","title":"Provided courthouse does not exist","status":400,"detail":"Courthouse 'TEST' not found."}""";
```

Comment:
The intent of the `type` field is really to allow consumers to identify specific errors, and potentially handle them. Are we confident that none of our API consumers (frontend) are reliant on the value of the `type` field for error handling? I think one of the ACs on DMP-2039 should be that none of the responses are impacted, i.e this is a refactoring change rather than a functional change. What's your view @mestebanez?